### PR TITLE
Use the CFT devtools light image in CICD

### PR DESCRIPTION
**Pull request by MartinPetkov**
Fixes https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/issues/850

**Note**: This commit updates the required Terraform version to 0.14.8

### CODE REVIEW

File: build/presubmit.yaml [hunk 1]
```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Terraform version used is 0.14.7.
+# Terraform version used is 0.14.8.
+substitutions:
+  _CFT_TOOLS_CONTAINER: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:8b0d85766044b0e4701729f9e70ee78dcecdb8688bd86915eb2a1cb00580ba1e"
```
Comment (xingao267) [inline build/presubmit.yaml:17]: does the light image work for our own CICD as well?

File: templates/tfengine/components/cicd/configs/tf-apply.yaml [hunk 2]
```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Note: Terraform version used in the automation is 0.14.7.
-{{- $cft := "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:a9f8f41fdd1f2e1919a052df23d5a15b07ecd29bb447d5cd111d5a80228f1e75"}}
+# Note: Terraform version used in the automation is 0.14.8.
```
Comment (xingao267) [inline templates/tfengine/components/cicd/configs/tf-apply.yaml:15]: also update the versions in README files for 3 binaries

**Review by xingao267 — COMMENTED** [review]
could you verify the three cloud build jobs work properly with a full deployment? a folder foundation deployment should be enough

File: build/presubmit.yaml (hunk 1 above)
Comment (MartinPetkov) [reply build/presubmit.yaml:17]: No, it's missing `go`.

**Comment by xingao267** [comment]
you can also use the testing project in go/dpt-tracker to test this change by updating the DPT version pin in the config for that folder foundation deployment

File: templates/tfengine/components/cicd/configs/tf-apply.yaml (hunk 2 above)
Comment (MartinPetkov) [reply templates/tfengine/components/cicd/configs/tf-apply.yaml:15]: Done.

**Review by xingao267 — APPROVED** [review]

**Comment by MartinPetkov** [comment]
Works fine when I tested it in the internal environment

**Pull request merged**